curvefs/proto/metaserver.proto

```diff
@@ -239,6 +239,12 @@ message CreateRootInodeResponse {
     optional uint64 appliedIndex = 2;
 }
 
+enum InodeOpenStatusChange {
+    OPEN = 1;
+    CLOSE = 2;
+    NOCHANGE = 3;
```

Comment:
NOCHANGE is used for what？

Reply:
MetaStatusCode UpdateInode(const Inode &inode,
                                       InodeOpenStatusChange statusChange =
                                           InodeOpenStatusChange::NOCHANGE) = 0;
UpdateInode adds a parameter InodeOpenStatusChange to update the InodeOpenStatusChange in UpdateInodeRequest，NOCHANGE is the default parameter.